scenarioo-server/src/main/java/org/scenarioo/business/diffViewer/comparator/ScreenshotComparator.java

```diff
@@ -88,7 +85,8 @@ public double compare(final String baseUseCaseName, final String baseScenarioNam
 		}
 
 		try {
-			int diffColor = new Color(255, 0, 0, 200).getRGB();
+			//int diffColor = new Color(255, 0, 0, 200).getRGB();
+			int diffColor = parameter.getDiffImageColor().getRGB();
```

Comment:
funny thing here --- so we are transforming a string to a BigInteger and then into an AWT color - just to in the end get out an int of it again? Couldnt we just use string for the color everywhere, and only here in this place we transform it to an Integer by using Integer.decode ??